nncf/openvino/graph/metatypes/openvino_metatypes.py

```diff
@@ -99,7 +104,8 @@ class OVGroupConvolutionBackpropDataMetatype(OVOpMetatype):
     name = "GroupConvolutionBackpropDataOp"
     op_names = ["GroupConvolutionBackpropData"]
     hw_config_names = [HWConfigOpName.CONVOLUTION]
-    const_channel_axis = [0, 2]  # const layout: [GROUPS, C_IN / GROUPS,  C_OUT / GROUPS, Z, Y, X]
+    const_channel_axis = [0, 2]
+    const_layout = [OVLayoutElem.GROUPS, OVLayoutElem.C_IN, OVLayoutElem.C_OUT]
```

Comment:
It looks like you add new abstraction but don't remove the previous representation `const_channel_axis`. What's the plan here?

Reply:
I've removed `const_channel_axis` for each metatype which have `const_layout`